src/ducks/transactions/TransactionsPage.jsx

```diff
@@ -180,14 +180,18 @@ class TransactionsPage extends Component {
          in the future, we check if the chosen month is before or after the
          current month.
       */
-      const monthsWithOperations = uniq(transactions.map(x => getMonth(x.date)))
-        .sort()
-        .reverse()
+      const monthsWithOperations = uniq(
+        transactions.map(x => getMonth(x.date))
+      ).sort()
       const beforeChosenMonth = x => x < month
       const afterChosenMonth = x => x > month
       const inRightDirection =
         month < this.state.currentMonth ? beforeChosenMonth : afterChosenMonth
-      const found = find(monthsWithOperations, inRightDirection)
+      const orderedMonthsWithOperations =
+        month < this.state.currentMonth
+          ? monthsWithOperations.reverse()
+          : monthsWithOperations
+      const found = find(orderedMonthsWithOperations, inRightDirection)
```

Comment:
I used find and findLast in SelectDates. I guess this code is no longer required...

https://github.com/cozy/cozy-banks/blob/master/src/ducks/filters/SelectDates.jsx#L46-L52

I think `find` and `findLast` makes it clearer and it is one array less to allocate (not that it is a problem) ;) What do you think ?

Reply:
You are right, it is better to avoid an extra array allocation.